lib/hq/graphql/enum.rb

```diff
@@ -49,7 +49,7 @@ def self.with_model(
       lazy_load do
         records = scope ? klass.instance_exec(&scope) : klass.all
         records.each do |record|
-          value "#{prefix}#{record.send(value_method).delete(" ")}", value: record
+          value "#{prefix}#{record.send(value_method).sub(/^((::)?\w+)::/, "").delete(" ")}", value: record
```

Comment:
It doesn't look like it removes spaces anymore. What are you trying to remove? Semi-colons? Anything else?

Reply:
It looks like this is breaking a handful of specs now. Are you running them locally? We don't have github actions setup yet to run them in CI

Reply:
I was having trouble with the colons.. the delete(" ") doesn't remove the spaces?

Reply:
I didn't see `delete(" ")` still on the end 🤦‍♂️.

Can this just be changed to this?
```suggestion
          value "#{prefix}#{record.send(value_method).delete(": ")}", value: record
```